src/software/ai/hl/stp/play/ball_placement_play_test.py

```diff
@@ -81,22 +86,174 @@ def test_two_ai_ball_placement(simulated_test_runner, run_enemy_ai):
         ),
     )
 
-    # Always Validation
-    always_validation_sequence_set = [[]]
+    # Placement Always Validation
+    placement_always_validation_sequence_set = [[]]
 
-    # Eventually Validation
-    eventually_validation_sequence_set = [
+    # Placement Eventually Validation
+    placement_eventually_validation_sequence_set = [
         [
             # Ball should arrive within 5cm of placement point
-            BallEventuallyEntersRegion(regions=[tbots.Circle(ball_final_pos, 0.05)]),
+            BallEventuallyEntersRegion(regions=[tbots.Circle(ball_final_pos, 0.1)]),
+            RobotEventuallyEntersRegion(regions=[tbots.Circle(ball_final_pos, 0.1)]),
         ]
     ]
 
     simulated_test_runner.run_test(
-        eventually_validation_sequence_set=eventually_validation_sequence_set,
-        always_validation_sequence_set=always_validation_sequence_set,
-        test_timeout_s=15,
+        eventually_validation_sequence_set=placement_eventually_validation_sequence_set,
+        always_validation_sequence_set=placement_always_validation_sequence_set,
+        test_timeout_s=TEST_DURATION,
+    )
+
+    # TODO (#2797): uncomment tests below to verify robots actually drop ball and exit region
+    # send a free kick command
+    # simulated_test_runner.gamecontroller.send_ci_input(
+    #     gc_command=Command.Type.DIRECT_FREE_BLUE, team=Team.BLUE
+    # )
+
+    # # Drop Ball Always Validation
+    # drop_ball_always_validation_sequence_set = [
+    #     [
+    #         BallAlwaysStaysInRegion(
+    #             regions=[tbots.Circle(ball_final_pos, 0.1)]),
+    #     ]
+    # ]
```

Comment:
Why does the ball have to always stay at its final position?

Reply:
So this is part of multistage pytests. The first stage tests that we get the ball to the destination, the second stage tests that we move away from the ball (the distance we move away from the ball is dependent on game state)